src/services/action/utils/export.test.ts

```diff
@@ -56,7 +56,7 @@ const setUpActions = async (app, member: Member) => {
     members: [member],
     itemMemberships: [itemMembership],
     item,
-    descendants: [getDummyItem()],
+    descendants: [FolderItemFactory() as unknown as Item],
```

Comment:
It is possible to use generic types here ? Maybe it could allow to avoid casting ? Maybe not...